partner_financial_risk/models/res_partner.py

```diff
@@ -62,32 +62,54 @@ def _compute_risk_allow_edit(self):
         is_editable = self.env.user.has_group(
             'base.group_sale_manager') or self.env.user.has_group(
             'account.group_account_manager')
-        for partner in self:
+        for partner in self.filtered('customer'):
             partner.risk_allow_edit = is_editable
 
```

Comment:
I'm still worried about "Credit limit" field is not editable when user has sale manager or account adviser rights.